### ISSUE TITLE
Highlighting of source code sections seems not to work for PDF

### ISSUE THREAD

**Issue by dietmarw** [xogeny/ModelicaBook#19]
I noticed that highlighted source code lines will not appear highlighted in the generated LaTeX/PDF version. I looked at the `writer/latex.py` from Sphinx where this functionality is actually present as of version 1.2.

So something to look into at some point.

**Comment by dietmarw** [comment]
Not likely go get fixed soon. So bumping to the next milestone so we can revisit it again.

**Comment by xogeny** [comment]
Seems like this is beyond our control, no?  So it really doesn't need to be in our issue tracker if it is already in theirs.  Presumably once there is an upstream fix, it will eventually trickle down to us.  Or do you think this is simple enough that we could somehow fix it?

**Comment by dietmarw** [comment]
It might be beyond our control but we still need a reminder to check the newer versions of sphinx in case this has been fixed so that we can update the docker configuration. So I suggest to keep this ticket and just move it from milestone to milestone after checking.

**Comment by dietmarw** [comment]
This will be fixed for Sphinx 1.6.6 though there is currently just a [small issue that is being worked on](https://github.com/sphinx-doc/sphinx/issues/1238#issuecomment-351315291). Once that is fixed we can update the book-builder to use 1.6.6 and configure the background colour to the same as the HTML uses
```
# Set line highlight background to light yellow
    'sphinxsetup': 'VerbatimHighlightColor={rgb}{1,1,0.8}',
```
:smile:

**Comment by dietmarw** [comment]
So this is now already fixed in the stable development branch. We now just need to wait for the 1.6.6 release to update the xogeny/book-builder and then the PDFs will als have highlighted lines.
